src/components/sbb-dialog/sbb-dialog.tsx

```diff
@@ -313,8 +322,9 @@ export class SbbDialog implements ComponentInterface {
       this._windowEventsController?.abort();
       this._focusTrap.disconnect();
       this._dialogContentResizeObserver.disconnect();
-      // Enable scrolling for content below the dialog
-      this._scrollHandler.enableScroll();
+      this._removeInstanceFromGlobalCollection();
+      // Enable scrolling for content below the dialog if no dialog is open
+      !dialogRefs.length && this._scrollHandler.enableScroll();
```

Comment:
For what scenario is this check needed?

Reply:
Without the check, after you close a stacked dialog the scroll on the page will be enabled again even if there is another dialog or dialogs still open.

Reply:
Ah I see. Was misreading once more the "&&" which calls a function and is not just a check...